### ISSUE TITLE
React: Masthead currently selected megamenu option is not read for screen reader users.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#4173]
## Detailed description

Issue: The currently selected megamenu is visually denoted with colour but the same information is not available for the screen reader users.

Expected:
The information conveyed visually should be made available for the screen reader users as well.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/a4ab2f5b-6880-419a-894f-82fdbc48de56)
